src/item.cpp

```diff
@@ -197,11 +197,15 @@ bool Item::equals(const Item* otherItem) const
 	}
 
 	if (!attributes) {
-		return !otherItem->attributes;
+		return otherItem->attributes ? (otherItem->attributes->attributeBits == 0) : true;
```

Comment:
maybe:
`return !otherItem->attributes || otherItem->attributes->attributeBits == 0`

Reply:
True, both do the same but yours is better.